umh-core/pkg/config/config.go

```diff
@@ -32,12 +32,13 @@ type FullConfig struct {
 }
 
 type InternalConfig struct {
-	Services       []S6FSMConfig          `yaml:"services,omitempty"`       // Services to manage, can be updated while running
-	Benthos        []BenthosConfig        `yaml:"benthos,omitempty"`        // Benthos services to manage, can be updated while running
-	Nmap           []NmapConfig           `yaml:"nmap,omitempty"`           // Nmap services to manage, can be updated while running
-	Redpanda       RedpandaConfig         `yaml:"redpanda,omitempty"`       // Redpanda config, can be updated while running
-	BenthosMonitor []BenthosMonitorConfig `yaml:"benthosMonitor,omitempty"` // BenthosMonitor config, can be updated while running
-	Connection     []ConnectionConfig     `yaml:"connection,omitempty"`     // Connection services to manage, can be updated while running
+	Services        []S6FSMConfig          `yaml:"services,omitempty"`        // Services to manage, can be updated while running
+	Benthos         []BenthosConfig        `yaml:"benthos,omitempty"`         // Benthos services to manage, can be updated while running
+	Nmap            []NmapConfig           `yaml:"nmap,omitempty"`            // Nmap services to manage, can be updated while running
+	Redpanda        RedpandaConfig         `yaml:"redpanda,omitempty"`        // Redpanda config, can be updated while running
+	BenthosMonitor  []BenthosMonitorConfig `yaml:"benthosMonitor,omitempty"`  // BenthosMonitor config, can be updated while running
+	Connection      []ConnectionConfig     `yaml:"connection,omitempty"`      // Connection services to manage, can be updated while running
+	RedpandaMonitor *RedpandaMonitorConfig `yaml:"redpandaMonitor,omitempty"` // RedpandaMonitor config, can be updated while running
```

Comment:
why is this a pointer? the other components are not a pointer?

Reply:
This allows to easily remove the redpanda monitor from s6.
BenthosMonitor & Connection are both also ptr types, the difference being that there is only ever 1 redpanda monitor.